rclcpp/include/rclcpp/node_interfaces/node_graph.hpp

```diff
@@ -67,6 +67,10 @@ class NodeGraph : public NodeGraphInterface
   std::map<std::string, std::vector<std::string>>
   get_service_names_and_types() const override;
 
+  RCLCPP_PUBLIC
+  std::map<std::string, std::vector<std::string>>
+  get_action_names_and_types(rclcpp::Node action_node) const override;
```

Comment:
`action_node` is never used? and `NodeGraph` possesses the `node_base_` in private scope?

```suggestion
  get_action_names_and_types() const override;
```